### DISCUSSION TITLE
interactive query with case-insensitive

### DISCUSSION THREAD

**Discussion by TornaxO7** [ajeetdsouza/zoxide#756] [Q&A]
Hello! I'd like to enable case-insensitive for interactive queries but I couldn't find how to do that.

Here's a minimal example of what I mean:
```
cd /tmp
mkdir Yes
zoxide add Yes
zoxide query --interactive # afterwards, enter `yes` => I expect to see `/tmp/Yes` listed here.
```
is that somehow achieveable?

**Comment by ajeetdsouza** [comment]
![image](https://github.com/ajeetdsouza/zoxide/assets/1777663/1f9f0edb-5b43-4e57-a5df-63a080b8c0f7)
Works fine for me. Have you set `_ZO_FZF_OPTS`?
